alpha-build-machinery/make/lib/golang.mk

```diff
@@ -1,6 +1,9 @@
 GO ?=go
 GOPATH ?=$(shell $(GO) env GOPATH)
-GO_PACKAGE :=$(subst $(GOPATH)/src/,,$(abspath .))
+gopath_list :=$(subst :, ,$(strip $(GOPATH)))
+# Use every path in GOPATH to try to remove it as a prefix of current dir to determine the package name.
+# If the prefix is not removed on subtitution, filter-out unchanged paths.
+GO_PACKAGE ?=$(strip $(filter-out $(abspath .),$(foreach p,$(gopath_list),$(patsubst $(p)/src/%,%,$(abspath .)))))
```

Comment:
do we really have to support all items in the GOPATH list or just the first?

Reply:
we are not guaranteed to be in the first one and it's seems negligible since we already have to do the hard part of splitting the gopath and substituting it